src/containers/tutor-home-page/add-photo-step/AddPhotoStep.jsx

```diff
@@ -4,7 +4,7 @@ import { style } from '~/containers/tutor-home-page/add-photo-step/AddPhotoStep.
 
 const AddPhotoStep = ({ btnsBox }) => {
   return (
-    <Box sx={style.root}>
+    <Box data-testid='AddPhotoStepContainer' sx={style.root}>
```

Comment:
Agree with @OliaHrytsak, use `findByText` or mock `Box` component in test.

Reply:
Mocked the Box component, the test result is as followed:

 ✓ tests/unit/containers/tutor-home-page/add-photo-step/AddPhotoStep.spec.jsx  (1 test) 21ms
SonarQube report written to /Users/olgaiaremchuk/Desktop/spaceTostudy/frontEnd/test-report.xml
Test Files  1 passed (1)
Tests  1 passed (1)
Start at  01:09:39
Duration  1.29s